Gordon360/Models/CCT/MembershipView.cs

```diff
@@ -12,7 +12,6 @@ namespace Gordon360.Models.CCT
     public partial class MembershipView
     {
         public int MembershipID { get; set; }
-        [Required]
```

Comment:
Do you know why several of these columns on MembershipView are no longer required? Presumably that represents a DB change.